app/controllers/refinery/blog/admin/posts_controller.rb

```diff
@@ -86,7 +86,8 @@ def post_params
       protected
 
         def find_post
-          @post = Refinery::Blog::Post.find_by_slug_or_id(params[:id])
+          @post = Refinery::Blog::Post.find_by_slug(params[:id]) if params[:id].friendly_id?
+          @post = Refinery::Blog::Post.find(params[:id]) if @post.nil?
```

Comment:
To use "config.use :finders"
Dne 20.2.2015 8:54 "Prokop Simek" <gilotka@gmail.com> napsal(a):

> Hi, I preffer friendly_id initializer...
> Dne 20.2.2015 5:21 "Philip Arndt" <notifications@github.com> napsal(a):
>
>> In app/controllers/refinery/blog/admin/posts_controller.rb
>> <https://github.com/refinery/refinerycms-blog/pull/404#discussion_r25048505>
>> :
>>
>> > @@ -86,7 +86,8 @@ def post_params
>> >        protected
>> >
>> >          def find_post
>> > -          @post = Refinery::Blog::Post.find_by_slug_or_id(params[:id])
>> > +          @post = Refinery::Blog::Post.find_by_slug(params[:id]) if params[:id].friendly_id?
>> > +          @post = Refinery::Blog::Post.find(params[:id]) if @post.nil?
>>
>> I think that ultimately this should just be:
>>
>> @post = Refinery::Blog::Post.friendly.find(params[:id])
>>
>> FriendlyId handles this internally.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/refinery/refinerycms-blog/pull/404/files#r25048505>.
>>
>